changelog.md

```diff
@@ -1,5 +1,9 @@
 # Next
 
+# 0.6.NEXT
```

Comment:
Oh I see I added changelog here, not the other PR. But nevermind, we're gonna hopefully merge both :)